tox.ini

```diff
@@ -54,3 +55,24 @@ commands: {[common-lint]commands}
 [testenv:py{37,38,39,310,311}-lint]
 extras: {[common-lint]extras}
 commands: {[common-lint]commands}
+
+[testenv:py{37,38,39,310,311}-wheel]
```

Comment:
I vote to copy `web3.py` here and we should test it with each repo. I had issues just using `build` as the dep there. I had to use `build[virtualenv]`.

Maybe...

```ini
deps=
    wheel
    build[virtualenv]
allowlist_externals=
    bash.exe
commands=
    bash.exe -c "rm -rf build dist"
    python -m build
    bash.exe -c 'pip install --upgrade "$(ls dist/<MODULE_NAME>-*-py3-none-any.whl)" --progress-bar off'
    python -c "import <MODULE_NAME>"
skip_install=true
```

Reply:
Right now there's just the unix wheel build. Should we add a windows wheel test to the template too?

Reply:
Discussed externally, wheel-windows ci run added for py311